public/VectorPairHMM/src/main/c++/utils.cc

```diff
@@ -32,6 +32,9 @@ uint8_t ConvertChar::conversionTable[255];
 //Global function pointers in utils.h
 float (*g_compute_full_prob_float)(testcase *tc, float* before_last_log) = 0;
 double (*g_compute_full_prob_double)(testcase *tc, double* before_last_log) = 0;
+#if defined(__POWER8_VECTOR__)
+unsigned long g_max_num_threads = omp_get_num_procs() *3/8; // 3 threads per core
```

Comment:
Could you please explain this line? Why do you assume there are 8 threads per core?